pkgs/development/php-packages/datadog_trace/default.nix

```diff
@@ -1,22 +1,56 @@
-{ buildPecl, curl, fetchFromGitHub, lib, pcre2, php }:
+{ lib
+, stdenv
+, php
+, cargo
+, rustc
+, fetchFromGitHub
+, rustPlatform
+, curl
+, pcre2
+, libiconv
+, darwin
+}@attrs:
 
-buildPecl rec {
-  pname = "ddtrace";
-  version = "0.82.0";
+php.buildPecl rec {
+  stdenv = if attrs.stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else attrs.stdenv;
```

Comment:
That is not necessary. darwin.apple_sdk_11_0.stdenv is stdenv for linux.

Reply:
When I do that, I get the following:

```
❯ nix build .#php.extensions.datadog_trace -L
warning: Git tree '/home/devlin/Code/NixOS/nixpkgs' is dirty
error: Package ‘apple-framework-CoreFoundation-11.0.0’ in /nix/store/pnjm969l5vsskflcydqik3ahbch3inhh-source/pkgs/os-specific/darwin/apple-sdk-11.0/apple_sdk.nix:77 is not available on the requested hostPlatform:
         hostPlatform.config = "x86_64-unknown-linux-gnu"
         package.meta.platforms = [
           "x86_64-darwin"
           "i686-darwin"
           "aarch64-darwin"
           "armv7a-darwin"
         ]
         package.meta.badPlatforms = [ ]
       , refusing to evaluate.

       a) To temporarily allow packages that are unsupported for this system, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1

        Note: For `nix shell`, `nix build`, `nix develop` or any other Nix 2.4+
        (Flake) command, `--impure` must be passed in order to read this
        environment variable.

       b) For `nixos-rebuild` you can set
         { nixpkgs.config.allowUnsupportedSystem = true; }
       in configuration.nix to override this.

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
         { allowUnsupportedSystem = true; }
       to ~/.config/nixpkgs/config.nix.
(use '--show-trace' to show detailed location information)
~/C/N/nixpkgs > php/extensions/bump-datadog-trace +1 -1 [!] ✘ 
```